src/compile/data/optimizers.ts

```diff
@@ -5,21 +5,34 @@ import {ParseNode} from './formatparse';
 import {SourceNode} from './source';
 import {TimeUnitNode} from './timeunit';
 
+export interface OptimizerFlags {
+  /**
+   * If true iteration continues
+   */
+  continueFlag: boolean;
```

Comment:
Let's rename these to just `continue` and `mutated` in a follow up PR. Also, add a comma to the comment: "If true, iteration continues."

Reply:
`continue` is a reserved keyword in javascript.

Reply:
Ups... then just fix the comment.